src/invert/fft_fftw.cxx

```diff
@@ -156,8 +156,9 @@ void rfft(MAYBE_UNUSED(const BoutReal *in), MAYBE_UNUSED(int length), MAYBE_UNUS
   const int nmodes = (n/2) + 1;
 
   // Store the output in out, and normalize
-  for(int i=0;i<nmodes;i++)
-    out[i] = dcomplex(fout[i][0], fout[i][1]) * fac; // Normalise
+  for (int i = 0; i < nmodes; i++) {
+    out[i] = dcomplex(fout[i][0], fout[i][1]) * fac;
```

Comment:
warning: do not use pointer arithmetic [cppcoreguidelines-pro-bounds-pointer-arithmetic]

```cpp
out[i] = dcomplex(fout[i][0], fout[i][1]) * fac;
    ^
```